### PR TITLE
Added initial support of slash commands

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: .github/workflows/help-command.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - name: Update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          token: ${{ secrets.NEPTUNE_BOT_ACCESS_TOKEN }}
+          comment-id: ${{ github.event.client_payload.github.payload.comment.id }}
+          body: |
+            > Command | Description
+            > --- | ---
+            > /pre-commit-apply | Commit a pre-commit suggestions
+            > /run-unit | Run unittests on full matrix
```
Comment (PatrykGala) [inline .github/workflows/help-command.yml:20]: `/run-unit` - unit tests should run on each commit. Do we need command?

File: .github/workflows/help-command.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            > /run-e2e | Run E2E tests on full matrix
```
Comment (PatrykGala) [inline .github/workflows/help-command.yml:21]: `/run-all-test` = `run-e2e + run-integrations` ?

**Review by PatrykGala — APPROVED** [review]

File: .github/workflows/help-command.yml (hunk 1 above)
Comment (Raalsky) [reply .github/workflows/help-command.yml:20]: On each commit we're running ubuntu x py3.7 + py.310 and that command will trigger unittests on full matrix (win,mac,linux x 3.7-3.10).

File: .github/workflows/help-command.yml (hunk 2 above)
Comment (Raalsky) [reply .github/workflows/help-command.yml:21]: Sure :wink: as you wish

File: .github/workflows/help-command.yml (hunk 1 above)
Comment (PatrykGala) [reply .github/workflows/help-command.yml:20]: Can you add the matrix to description? 
```
/run-unit | Run unittests on full matrix*
/run-e2e | Run E2E tests on full matrix*
*(win,mac,linux x 3.7-3.10)
```

**Review by PatrykGala — APPROVED** [review]

**Review by PatrykGala — APPROVED** [review]

**Pull request merged**